package.json

```diff
@@ -5,17 +5,20 @@
   "type": "module",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "nodemon app.js"
+    "start": "nodemon app.js",
+    "env": "node --env-file=.env app.js"
```

Comment:
you have to create a single command to run the application and load .env file. 
You can use --env-file parameter or dotenv package that you installed as well